ui/src/Components/Dashboard/TableRow/TableRow.js

```diff
@@ -0,0 +1,19 @@
+import Toggle from "../../Global/Toggle/Toggle"
+
+export default function TableRow({ featureFlag, headers }) {
+  return (
+    <tr
+      className='border-t'
+    >
+      {headers.map((heading, index) =>
+        <td className='px-4 py-2' key={index}>{featureFlag[heading]}</td>
+      )}
+      <td className='px-4 py-2'>
+        <Toggle 
+          active={featureFlag.enabled}
+        />
+      </td>
```

Comment:
This is the Configuration column of the table. Will include toggle, edit & delete button. Should I abstract it in a different component?